src/Repository/TicketRepository.php

```diff
@@ -28,4 +29,38 @@ public function findAllOrderByStatusAndCreationDate()
 
         return $query->getResult();
     }
+
+    /**
+     * @param int $delay
+     * @return QueryBuilder
+     * @throws \Exception
+     */
+    public function bulkUpdateContactTicketCloseAfterDelay(int $delay)
+    {
+        $queryBuilder = $this->ticketClosedAfterDelayQueryBuilder($delay);
+        return $queryBuilder
+            ->update()
+            ->set('t.contact', ':contact')
+            ->setParameter('contact', null)
```

Comment:
Le champ contact n'est pas nullable, on le rend nullable ou on met un texte vide ?

Reply:
As you want :)